cmake/modules/AddSwift.cmake

```diff
@@ -409,10 +418,9 @@ endfunction()
 # LLVM_LINK_COMPONENTS
 #   LLVM components this library depends on.
 #
-# IGNORE_LLVM_UPDATE_COMPILE_FLAGS
-#   If set do not use llvm_update_compile_flags to generate cflags/etc.  This is
-#   generally used when compiling a mixed c/c++/swift library and one wants to
-#   disable this for the swift part.
+# PURE_SWIFT
+#   This has two effects if set: we do not use llvm_update_compile_flags to
+#   generate cflags/etc and we leave the linking mode of the library as swift.
```

Comment:
To be clear, I am going to update llvm_update_compile_flags upstream once I get ErikE up and running with libswift. So the llvm_update_compile_flag part of this will go away with time.